carp.studies.core/src/jvmTest/kotlin/dk/cachet/carp/studies/infrastructure/versioning/OutputRecruitmentServiceTestRequests.kt

```diff
@@ -0,0 +1,21 @@
+package dk.cachet.carp.studies.infrastructure.versioning
+
+import dk.cachet.carp.common.test.infrastructure.versioning.OutputTestRequests
+import dk.cachet.carp.studies.application.RecruitmentService
+import dk.cachet.carp.studies.application.RecruitmentServiceHostTest
+import dk.cachet.carp.studies.application.RecruitmentServiceTest
+import dk.cachet.carp.studies.application.StudyService
+import dk.cachet.carp.studies.infrastructure.RecruitmentServiceLog
+
+
+private val services = RecruitmentServiceHostTest.createService()
+
+class OutputProtocolServiceTestRequests :
+    OutputTestRequests<RecruitmentService>(
+        RecruitmentService::class,
+        RecruitmentServiceLog( services.first, services.third )
```

Comment:
Documentation is needed when accessing generic tuple indexes like this I think.

Reply:
Agreed. Replaced with data classes; a case where I do find self-documenting code is sufficient.